cfgov/unprocessed/css/organisms/footer.less

```diff
@@ -202,6 +209,10 @@
                 padding-right: @grid_gutter-width;
             }
         } );
+
+        .respond-to-print({
```

Comment:
Should we aggregate all the print styles at the bottom instead of sprinkling them throughout?

Reply:
Good question. I was following the lead of the other `respond-to-` media query mixins in the file, which seems like the right path for now. If we want to refactor all of those mixins later, I'm open to it.

Reply:
For what it's worth, in [mega-menu.less](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/css/organisms/mega-menu.less#L709) the queries are aggregated together in distinct sections of the file and I find it a lot easier to work with. You can look at one screen size and go to one section of the less file to make edits in most cases. The disadvantage is if a selector is deleted, it may need to be deleted in each media query block. However, that's worth it for the one-to-one relationship of one code block to one media state, in my opinion.

Reply:
Yeah, interesting. Seems like it'd be a good thing to discuss during a FEWD bi-weekly.